src/indexset.jl

```diff
@@ -557,11 +511,11 @@ Specify the number of output indices with `Order(N)`.
 """
 function Base.setdiff(f::Function,
                       ::Order{N},
```

Comment:
The reason for these functions that could be called with `Order` inputs was to help with type stability when you knew the length of the result. However, with the new design that isn't necessary (other than adding a runtime check for the length of the result, which I don't think is really necessary to have). So versions of functions that accept `Order` probably aren't necessary anymore. I doubt people were using them anyway, but I guess we can deprecate them.

Reply:
Sure :)